ms2pip/cython_modules/ms2pip_peaks_c.c

```diff
@@ -9,23 +9,25 @@
 #include "ms2pip_features_c_catboost.c"
 
 #include "../models/CID.h"
-#include "../models/HCD.h"
+#include "../models/HCD-2019.h"
+// #include "../models/HCD-2021.h"
+#include "../models/HCD-2021-fast.h"
```

Comment:
I guess the 2021 imports, and headers and C files, can be dropped if we use the xgboost models anyway? Otherwise, the HCD-2021 model should be added to the various functions.